modules/phase_field/src/meshgenerators/EBSDMeshGenerator.C

```diff
@@ -0,0 +1,164 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "EBSDMeshGenerator.h"
+#include "MooseApp.h"
+
+#include <fstream>
+
+registerMooseObject("PhaseFieldApp", EBSDMeshGenerator);
+
+InputParameters
+EBSDMeshGenerator::validParams()
+{
+  InputParameters params = GeneratedMeshGenerator::validParams();
+  params.addClassDescription("Mesh generated from a specified DREAM.3D EBSD data file.");
+  params.addRequiredParam<FileName>("filename", "The name of the file containing the EBSD data");
+  params.addParam<unsigned int>(
```

Comment:
Why do you have this? It is already available in Mesh/uniform_refine

Reply:
No. Mit with that updated documentation.

Reply:
Can you use setDocString instead: https://github.com/idaholab/moose/blob/01b19be81e97d49e48b700c787c652322a8e6459/framework/src/utils/InputParameters.C#L297

Reply:
No you can't, you use it in here.